SonarQube.TeamBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -70,6 +83,8 @@ public static AnalysisConfig GenerateFile(ProcessedArgs args, TeamBuildSettings
                 config.SetSettingsFilePath(args.PropertiesFileName);
             }
 
+            config.AnalyzerSettings = analyzerSettings;
```

Comment:
The previous commit tested that the config containing the analyser settings is XML-serializable, so to add the analyser settings to the file we just need to set the property.